api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -124,6 +146,11 @@ message ProcessingRequest {
   //   are needed.
   //
   bool observability_mode = 10;
```

Comment:
This ideally belongs inside ProtocolConfiguration right? Should/Could we move it?

Reply:
Yeah, ideally we should move it in. However that will be a breaking change.  I am open if everyone agrees to move it.

Reply:
You can not move it, I think. You can make this field as deprecated and add a new one in the ProtocolConfiguration with the doc on which field takes precedence.

Reply:
In that case, let's do not touch observability_mode in this PR.  We can do the deprecation/add-new in a separate PR to keep this PR clean.